src/components/card/DueDateSelector.vue

```diff
@@ -178,6 +178,10 @@ export default defineComponent({
 				: null
 			// Next Monday 08:00 AM
 			const nextWeekTime = moment().add(1, 'weeks').day(1).hour(8)
+			// Next Month 1st 08:00 AM
+			const nextMonthTime = moment().add(1, 'months').day(1).hour(8)
+			// Next Quarter 1set 08:00 AM
+			const nextQuarterTime = moment().add(3, 'months').day(1).hour(8)
```

Comment:
In line 182 and 184: the `.day(1)` will select the Monday but not the first date in month. It doesn't seem to match the comment.